src/arch/arm/kernel/boot.c

```diff
@@ -147,6 +147,9 @@ BOOT_CODE static void init_irqs(cap_t root_cnode_cap)
     setIRQState(IRQIPI, CORE_IRQ_TO_IRQT(getCurrentCPUIndex(), irq_reschedule_ipi));
 #endif
 
+    for (i = 0; i < ARRAY_SIZE(reserved_irqs); i++) {
+        setIRQState(IRQReserved, CORE_IRQ_TO_IRQT(0, reserved_irqs[i]));
```

Comment:
One issue with this approach is that the current gicv3 driver disables these IRQs at the distributor.  IT's possible for one kernel to disable IRQs of another kernel if it starts up later.